### PR TITLE
Best Practices for Secure and Fair workflows

### PR DESCRIPTION
**Pull request by coverbeck**

### CODE REVIEW

**Review by coverbeck — COMMENTED** [review]
See https://docs.dockstore.org/en/bestpractices/advanced-topics/best-practices/best-practices-secure-fair-workflows.html

The rendering of bulleted items is inconsistent; some are bold, some are not. FWIW, I think the non-bolded ones look better, but either way, as long as they are consistent.

**Comment by denis-yuen** [comment]
Getting better, links are still messed up in https://docs.dockstore.org/en/bestpractices/advanced-topics/best-practices/best-practices-secure-fair-workflows.html#interoperable

File: docs/advanced-topics/best-practices/best-practices-secure-fair-workflows.rst [hunk 1]
```diff
@@ -0,0 +1,251 @@
+Best Practices for Secure and FAIR workflows
```
Comment (coverbeck) [inline docs/advanced-topics/best-practices/best-practices-secure-fair-workflows.rst:1]: I think it should be "Workflows", with a capital W.

**Comment by bethsheets** [comment]
I fixed the bolding issues, but now the spacing is kinda funny. Should we add spacing between every bullet point or it is ok as is?

Also, linking to relative paths instead of https doesn't seem to show links to those docs?

**Comment by bethsheets** [comment]
I think the formatting issues and links are all corrected now.

**Review by NatalieEO — APPROVED** [review]
Looks really good, thanks Beth

**Review by denis-yuen — APPROVED** [review]
Nice! 
Sorry for all the formatting issues

**Review by bethsheets — APPROVED** [review]
Thanks for the reviews and help with RST, everyone!  Glad to get this out.

**Pull request merged**